src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { useEffect, useState, useMemo } from 'react';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PersonLink } from '../PersonLink';
+
+const preparePeopleWithLinks = (peopleList: Person[]) => {
+  return peopleList.map(person => {
+    const { fatherName, motherName } = person;
+
+    return {
+      ...person,
+      fatherNameLink:
+        peopleList.find(candidate => candidate.name === fatherName)?.slug ||
+        null,
+      motherNameLink:
+        peopleList.find(candidate => candidate.name === motherName)?.slug ||
+        null,
```

Comment:
```suggestion
      father:
        peopleList.find(candidate => candidate.name === fatherName) ||
        null,
      mother:
        peopleList.find(candidate => candidate.name === motherName) ||
        null,
```
You can just store father and mother objects, so it much easier to get values from it what you actually need